### ISSUE TITLE
Feature Request: HiDpi scaling

### ISSUE THREAD

**Issue by jonseppanen** [vinceliuice/Colloid-gtk-theme#27]
Please enable hidpi scaling, similar to https://www.gnome-look.org/p/1289376/ and Elementary OS 6 theme.

This allows for fractional ui scaling based on dpi instead of resolution, allowing for a superior experience over any other OS.

Thanks

**Comment by vinceliuice** [comment]
Isn't this a system feature? all gtk theme can scaling with system hipi setting

**Comment by jonseppanen** [comment]
No, the gtk.css needs to have static pixel scaling values for elements changed to rem values, as per the supplied example.

**Comment by vinceliuice** [comment]
Could you show me some screenshot?, I don't know this issue
